server/rabbitmq.go

```diff
@@ -59,6 +63,7 @@ func (s *AMQPServer) NotifyConnected() chan bool {
 }
 
 func (s *AMQPServer) RegisterEndpoint(endpoint *Endpoint) {
+	endpoint.server = s
```

Comment:
Agreed. Arguably this should even `panic` as it would be executed on startup, and we want the error to not be ignored.